ui/imports/shared/popups/common/EnterSeedPhrase.qml

```diff
@@ -88,6 +88,13 @@ Item {
                     root.store.addAccountModule.newKeyPairName = text
                 }
 
+                validators: [
+                    StatusMinLengthValidator {
+                        errorMessage: qsTr("Keypair name must be at least %1 characters").arg(Constants.addAccountPopup.keyPairAccountNameMinLength)
```

Comment:
Here too

Reply:
Given the amount of repetition, extract the whole Validator somewhere? :)

Reply:
I'd vote for a AccountNameInput and KeypairNameInput.
Many other things are repeated
If you agree i ll create the issue